client/components/domains/domain-registration-suggestion/index.jsx

```diff
@@ -132,6 +132,109 @@ class DomainRegistrationSuggestion extends Component {
 		return includes( this.props.unavailableDomains, domain );
 	};
 
+	getSelectDomainAriaLabel() {
+		const { suggestion, translate, productCost, productSaleCost } = this.props;
+		const priceRule = this.getPriceRule();
+
+		const baseLabel = translate( 'Select domain %(domainName)s', {
+			args: { domainName: suggestion.domain_name },
+			context: 'Accessible label for domain selection button. %(domainName)s is the domain name.',
+		} );
+
+		switch ( priceRule ) {
+			case 'ONE_TIME_PRICE':
```

Comment:
Nit: I noticed the string literals in the switch, using constants might help with maintainability down the line. Totally get that it's not worth changing for this fix though. Just dropping a note in case it comes up again later!

Reply:
Thank you for the suggestion, Roberto! I agree it would make sense to use constants here. Looking through the codebase, there are already number of places where these string literals are being used. Replacing them with constants would help with maintainability indeed.

I will consider this for a separate PR as I don't want to add more changes to this one PR. 🙂